package.json

```diff
@@ -2,14 +2,25 @@
   "name": "@camunda/feel-builtins",
   "version": "0.0.1",
   "description": "Collection of builtin Camunda extensions for FEEL",
+  "type": "module",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "lint": "eslint .",
+    "update:builtins": "run-s pull:camunda-docs compile:builtins",
+    "pull:camunda-docs": "git -C camunda-docs pull || git clone git@github.com:camunda/camunda-docs.git camunda-docs",
+    "compile:builtins": "node scripts/compileBuiltins.mjs"
+  },
+  "dependencies": {},
+  "devDependencies": {
+    "eslint-plugin-bpmn-io": "^2.2.0",
+    "eslint": "^9.31.0",
+    "glob": "^11.0.3",
+    "marked": "^16.0.0",
+    "npm-run-all2": "^8.0.0"
   },
   "files": [
-    "dist"
+    "src"
   ],
-  "main": "dist/index.js",
-  "module": "dist/index.es.js",
+  "main": "src/index.js",
```

Comment:
Let's add a `"sideEffects": false` annotation, to allow tree-shaking in the future - https://github.com/bpmn-io/bpmn-js/blob/develop/package.json#L62.